massa-execution-exports/src/test_exports/settings.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+//! This file defines testing tools related to the config
+
+use massa_time::MassaTime;
+
+use crate::ExecutionConfig;
+
+/// Default value of ExecutionConfig used for tests
+impl Default for ExecutionConfig {
+    fn default() -> Self {
+        use massa_models::constants::default_testing::*;
+
+        Self {
+            readonly_queue_length: READONLY_QUEUE_LENGTH,
+            max_final_events: MAX_FINAL_EVENTS,
+            thread_count: THREAD_COUNT,
+            cursor_delay: CURSOR_DELAY,
+            clock_compensation: Default::default(),
+            // reset genesis timestamp because we are in test mode that can take a while to process
+            genesis_timestamp: MassaTime::now().expect("Impossible to reset the timestamp in test"),
+            t0: 10.into(),
```

Comment:
why ? in other tests, t0 is more around 500 ms

Reply:
Because I want to accelerate the tic tac for my tests since the execution of smart contract is really fast.

Reply:
:watch: